tools/pipelines/build-docs.yml

```diff
@@ -2,14 +2,6 @@
 # Licensed under the MIT License.
```

Comment:
Nit: since this pipeline is now about generating and publishing doc model artifacts, I wonder if we should give it a more specific name. E.g., `publish-api-model-artifact` or something?